### ISSUE TITLE
Login end with oops after 7.2.2 upgrade

### ISSUE THREAD

**Issue by ricardoteixeiraduarte** [bitium/jira-saml-plugin#46]
After upgrade to 7.2.2, /plugins/servlet/saml/auth shows "oops".
Full browser refresh is required to make jira work again.
Session is created tho.

**Comment by blt04** [comment]
This affected us as well.  One potential fix is: #47

**Comment by requenym** [comment]
+1 Having the same issue

**Comment by pearj** [comment]
I had a look at the upstream error in Jira itself: https://jira.atlassian.com/browse/JRA-63278 and so I tried removing the base URLs, but that made it mean that I had to log in twice, which was really odd.

So then I added a filter step before redirecting so that any attempt to redirect to `/secure/Dashboard.jspa` is instead `/default.jsp`.

It does mean that atlassian-saml needs to be modified too, so that jira can filter the redirect.
